src/pathfinding_service/api.py

```diff
@@ -430,6 +421,28 @@ def get(  # pylint: disable=no-self-use
                     dict(source=req["source"], target=req["target"], routes=req["routes"])
                 )
 
+        decoded_target_address: Optional[Address] = None
+        if target_address:
+            decoded_target_address = Address(to_canonical_address(target_address))
+
+        feedback_routes = list(
```

Comment:
Seems like you don't need `list` here. The loop below should work on an iterator as well and save some memory.

Reply:
Fixed